src/styles/blocks/events.scss

```diff
@@ -0,0 +1,95 @@
+.events {
+  &__title {
+    font-family: "Playfair Display", sans-serif;
+    font-weight: 700;
+    font-size: 28px;
+    line-height: 125%;
+    margin-bottom: 60px;
+
+    color: #1c1b29;
+
+    @include onTablet {
+      font-size: 40px;
+      margin-bottom: 75px;
+    }
+
+    @include onDesktop {
+      font-size: 48px;
+      line-height: 120%;
+      margin-bottom: 70px;
+    }
+  }
+
+  &__subtitle {
+    font-family: "Playfair Display", sans-serif;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 125%;
+
+    color: #1c1b29;
+
+    margin-bottom: 20px;
+  }
+
+  &__paragraph {
+    font-family: "Raleway", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 150%;
+
+    color: $color-red;
+    margin-bottom: 10px;
+
+    &-color {
+      color: #504f5e;
+      margin-bottom: 30px;
+
+      @include onTablet {
+        display: flex;
+        margin-bottom: 40px;
+      }
+
+      @include onDesktop {
+        margin-bottom: 50px;
+      }
+    }
+  }
+
+  &__photo-container {
+    position: relative;
+    width: 100%;
+    max-height: 475px;
+    margin-bottom: 20px;
+
+    @include onTablet {
+      height: 420px;
+      padding-bottom: 0;
+    }
+
+    @include onDesktop {
+      height: 484px;
+      padding-bottom: 0;
+    }
+  }
+
+  &__img {
+    // position: absolute;
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+    cursor: zoom-in;
+    transition: transform 0.3s, border-radius 0.3s;
+
+    &:hover {
+      transform: scale(1.05);
+      border-radius: 5%;
```

Comment:
why do you use percent with this style? Do you know what this percent is calculated from?